data_steward/analytics/cdr_ops/controlled_tier_qc/sql/query_templates.py

```diff
@@ -149,7 +149,7 @@
     post_deid.{{ column_name }} AS output_pid
 FROM `{{ project_id }}.{{ post_deid_dataset }}.{{ table_name }}` post_deid
 LEFT JOIN `{{ project_id }}.{{ pre_deid_dataset }}._mapping_{{ table_name|replace("_ext","") }}` pre_deid USING({{ primary_key }})
-LEFT JOIN `{{ project_id }}.{{ mapping_dataset }}.{{ mapping_table }}` map ON pre_deid.src_hpo_id = map.hpo_id
+LEFT JOIN `{{ project_id }}.pipeline_tables.{{ mapping_table }}` map ON pre_deid.src_hpo_id = map.hpo_id
 )
```

Comment:
```suggestion
LEFT JOIN `{{ project_id }}.{{ pipeline_dataset }}.{{ mapping_table }}` map ON pre_deid.src_hpo_id = map.hpo_id
)
```

Reply:
oh.. during the discussion about the ticket there were 2 ways discussed and I interpreted it as 'if going with using pipeline_tables, parameterization is not required.'.. let me make that change.